src/test/java/org/spongepowered/common/registry/CatalogTypeMethodTest.java

```diff
@@ -45,7 +45,7 @@
 import java.util.Optional;
 import java.util.Set;
 
-@RunWith(LaunchWrapperParameterized.class)
+//@RunWith(LaunchWrapperParameterized.class)
```

Comment:
Reminder to uncomment. Was adding too much time for gradle to figure out which tests to execute or not.